init-tools.sh

```diff
@@ -107,6 +107,8 @@ if [ ! -e $__PROJECT_JSON_FILE ]; then
     $__DOTNET_CMD restore "$__PROJECT_JSON_FILE" --packages $__PACKAGES_DIR --source $__BUILDTOOLS_SOURCE
  fi
 
- sh $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR
+ # On ubuntu 14.04, /bin/sh (symbolic link) calls /bin/dash by default.
```

Comment:
Yes. Right. default shebang (/bin/bash)  of ./init-tool.sh executes line 110 by removing 'sh'.